tutorials/plot_artifacts_correction_ica.py

```diff
@@ -76,20 +76,19 @@
 # Component properties
 # --------------------
 #
-# Let's take a closer look at three potential candidates for artifact-related
-# components: IC 12, IC 15 and IC 21
+# Let's take a closer at properties of first three independent components
```

Comment:
do they make any sense?

we should add the EEGLAB way of sorting the components by explained variance...

@jmontoyam